src/satosa/micro_services/processors/scope_remover_processor.py

```diff
@@ -0,0 +1,15 @@
+from ..attribute_processor import AttributeProcessorError
+from .base_processor import BaseProcessor
+
+class ScopeRemoverProcessor(BaseProcessor):
+    """
+    Removes the scope from a given attribute 
+    
+    """
+    def process(self, internal_data, attribute, **kwargs):
+        attributes = internal_data.attributes
+        new_values = []
+        for value in attributes.get(attribute, [None]):
+            if '@' in value:
+                new_values.append(value.split('@')[0])
+        attributes[attribute] = new_values
```

Comment:
This will remove any attribute values without a `@`, and leave only values that had a scope (but with the scope-part removed). This is not clear from the class name. 

If this was not intentional you only need to remove the `if '@' in value` clause (I'm ignoring the problem described in the comment above)
```python
for value in attributes.get(attribute, [None]):
  value_with_no_scope = value.split('@')[0]
  new_values.append(value_with_no_scope)
```